tooling/src/hypothesistooling/__main__.py

```diff
@@ -45,10 +45,11 @@ def task(fn):
 
 @task
 def lint():
-    os.chdir(tools.HYPOTHESIS_PYTHON)
-    pip_tool('flake8', 'src', 'tests', '--config', os.path.join(
-        tools.ROOT, '.flake8'
-    ))
+    pip_tool(
+        'flake8',
```

Comment:
Note: our current run requires `flake8-docstrings` to be installed too.

Reply:
That's fine. This runs in a virtualenv that installs everything from `tools.txt`